src/test/java/api/support/APITests.java

```diff
@@ -210,7 +213,15 @@
     patronGroupsFixture);
 
   protected final LoansFixture loansFixture = new LoansFixture(
+          servicePointsFixture);
+
+  protected final CheckOutFixture checkOutFixture = new CheckOutFixture(
           usersFixture, servicePointsFixture);
+  
+  protected final CheckInFixture checkInFixture = new CheckInFixture(
+          servicePointsFixture);
+  
+  protected final ChangeDueDateFixture changeDueDateFixture = new ChangeDueDateFixture();
```

Comment:
Please can the indentation of these parameters use the standard two-space indentation after a line continuation